Vagrantfile

```diff
@@ -156,6 +157,7 @@ end
 
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    config.vm.box_check_update = false
```

Comment:
Hmm. I don't think this is an issue, but can this be done in its own commit so we can revert it if it becomes a problem?